src/PrivateKey.cpp

```diff
@@ -294,3 +300,7 @@ Data PrivateKey::signZilliqa(const Data& message) const {
 void PrivateKey::cleanup() {
     std::fill(bytes.begin(), bytes.end(), 0);
 }
+
+Data PrivateKey::bytesWithHederaDerPrefix() const {
+    return concat(parse_hex(gHederaDerPrefix), this->bytes);
+}
```

Comment:
This is very chain-specific, I prefer not to have chain-specific logic in general PrivateKEy, PublicKey, etc. classes if possible.
This could be moved to one Hedera class.
Or if this is general DER stuff, let's not name Hedera.